readinglistmanager/datasource.py

```diff
@@ -103,14 +103,15 @@ def fdel(self):
 class OnlineSource(Source):
     def __init__(self, name, file, tableDict):
         super().__init__(name, file, "WEB-DL")
-        printResults("Checking DB data in %s" % (file), 2)
+        printResults("Checking DB data in %s" % (file,), 2)
         try:
             self._tableReadingListTitles = tableDict['ReadingLists']
             self._tableReadingListDetails = tableDict['ReadingListDetails']
             self._tableIssueDetails = tableDict['IssueDetails']
         except Exception as e:
+            print("Caught error in OnlineSource:", e)
```

Comment:
The printResults function includes printing to console with print(), so this is redundant when coupled with the following line.